client/app/components/ownership-service-modal/ownership-service-modal-service.factory.js

```diff
@@ -0,0 +1,95 @@
+(function() {
+  'use strict';
+
+  angular.module('app.components')
+    .factory('OwnershipServiceModal', OwnershipServiceFactory);
+
+  /** @ngInject */
+  function OwnershipServiceFactory($modal) {
+    var modalService = {
+      showModal: showModal
+    };
+
+    return modalService;
+
+    function showModal(serviceDetails) {
+      var modalOptions = {
+        templateUrl: 'app/components/ownership-service-modal/ownership-service-modal.html',
+        controller: OwnershipServiceModalController,
+        controllerAs: 'vm',
+        size: 'lg',
+        resolve: {
+          serviceDetails: resolveServiceDetails,
+          users: resolveUsers,
+          groups: resolveGroups
+        }
+      };
+      var modal = $modal.open(modalOptions);
+
+      return modal.result;
+
+      function resolveServiceDetails(CollectionsApi) {
+       // return serviceDetails;
+        var requestAttributes = [
+          'evm_owner.userid',
+          'miq_group.description'
+        ];
+        var options = {attributes: requestAttributes};
+
+        return CollectionsApi.get('services', serviceDetails.id, options);
+      }
+    }
+  }
+
+  /** @ngInject */
+  function resolveUsers(CollectionsApi) {
+    var options = {expand: 'resources', attributes: ['userid', 'name']};
+    return CollectionsApi.query('users', options);
+  }
+
+  /** @ngInject */
+  function resolveGroups(CollectionsApi) {
+    var options = {expand: 'resources', attributes: ['description']};
+    return CollectionsApi.query('groups', options);
+  }
+
+  /** @ngInject */
+  function OwnershipServiceModalController(serviceDetails, $state, $modalInstance, CollectionsApi, Notifications, users, groups) {
+    var vm = this;
+
+    vm.service = serviceDetails;
+    vm.saveOwnership = saveOwnership;
+    vm.users = users;
+    vm.groups = groups;
+
+    vm.modalData = {
+      'action': 'set_ownership',
+      'resource': {
+        'owner': {
+          'userid': vm.service.evm_owner && vm.service.evm_owner.userid || ''
+        },
+        'group': {
+          'description': vm.service.miq_group && vm.service.miq_group.description || ''
+        }
+      }
+    };
+    activate();
```

Comment:
I've seen this empty method in a bunch of other places too. 
Since it is a standard controller method, I suppose you can leave it there in case if you want to override it in the future.